### ISSUE TITLE
middleware/pprof child page link error.

### ISSUE THREAD

**Issue by flowerbin** [kataras/iris#975]
**Question 1:**

Go to  http://xxx/debug/pprof/ redirect to http://xxx/debug/pprof, resulting in the relative path error of the sub-page link.

`<a href="block?debug=1">block</a>` link to http://xxx/debug/block?debug=1, not http://xxx/debug/pprof/block?debug=1

**Question 2:**

26 of the lines in the https://github.com/kataras/iris/blob/master/middleware/pprof/pprof.go for routing parameters`ctx.Values().GetString("action")` will always be an empty string。Should be `ctx.Params().Get("action")` ?

**Comment by kataras** [comment]
Yes @flowerbin, make a PR and fix that, it should be `ctx.Params().Get("action")`, good eye!

**Comment by jjhesk** [comment]
how to setup this pprof?
